tests/Jobs/ProductModel/RetrieveProductModelJobTest.php

```diff
@@ -32,4 +34,30 @@ public function it_has_correct_tags_and_unique_id(): void
         $this->assertEquals(['code'], $job->tags());
         $this->assertEquals('code', $job->uniqueId());
     }
+
+    #[Test]
+    public function it_can_fail(): void
+    {
+        /** @var ProductModel $product */
+        $product = ProductModel::query()->create([
```

Comment:
I'd prefer the variable to be called `$productModel` & `$productModelData` to avoid confusion, like the rest of the tests